content/500-platform/550-platfrom-cli/commands.mdx

```diff
@@ -0,0 +1,229 @@
+---
+title: 'Commands'
+metaTitle: 'Platform CLI: Commands'
+tocDepth: 5
+---
+
+<TopBlock>
+
+This document describes the Prisma Data Platform integrated Prisma CLI commands, arguments, and options.
+
+</TopBlock>
+
+## Getting started
+
+To get started, ensure you have the [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) updated to version `5.7.0` or later. This is necessary to access the Platform through the Prisma CLI.
+
+<Admonition>
+
+💡 When using commands, always start with `prisma platform` and include the `--early-access` flag to enable the use of the Prisma Data Platform whilst still in early access.
+
+</Admonition>
+
+## Authentication
+
+### <inlinecode>platform</inlinecode>
+
+#### <inlinecode>auth login</inlinecode>
+
+Logs into your Prisma Data Platform account or creates a new one.
+
+```bash
+npx prisma platform auth login --early-access
+```
+
+#### <inlinecode>auth logout</inlinecode>
+
+Logs out of your Prisma Data Platform account.
+
+```bash
+npx prisma platform auth logout --early-access
+```
+
+#### <inlinecode>auth show</inlinecode>
+
+Displays information about the currently authenticated user.
+
+```bash
+npx prisma platform auth show --early-access
+```
+
+### Workspace Management
+
+### <inlinecode>platform</inlinecode>
+
+#### <inlinecode>workspace show</inlinecode>
+
+Lists all workspaces available to your account.
+
+```bash
+npx prisma platform workspace show --early-access
+```
+
+### Project Management
+
+### <inlinecode>platform</inlinecode>
+
+#### <inlinecode>project show</inlinecode>
+
+Lists all projects within the specified workspace.
+
+```bash
+npx prisma platform project show \
+--workspace cdlgqeDwSJxhazrsFgouRg2zQ \
+--early-access
+```
```

Comment:
Should the workspace id 'cdlgqeDwSJxhazrsFgouRg2zQ' be replaced with something like '[insert workspace-id]' so it's clear that it needs to be replaced with the actual workspace id?

Reply:
I like that suggestion and applied it to the docs 🙌